content/go/server.go.md

```diff
@@ -542,8 +542,8 @@ func printAsJsonFormat(data interface{}) {
 
 /*
 Cancel a bare metal server.
-The server will be cancelled immediately if it is billed Hourly,
-for Monthly server the cancellation will be made after next bill date.
+The server will be canceled immediately if it is billed Hourly,
```

Comment:
cancelled

Reply:
Since both options seem [valid](https://www.merriam-webster.com/words-at-play/canceled-or-cancelled), I keep the current one.

Reply:
English is a "fun" language lol.